iohk-monitoring/src/Cardano/BM/Output/LogBuffering.lhs

```diff
@@ -0,0 +1,83 @@
+\subsection{Cardano.BM.Output.LogBuffer}
+\label{module:Cardano.BM.Output.LogBuffer}
```

Comment:
This file shouldn't be deleted? It is the same with `iohk-monitoring/src/Cardano/BM/Output/LogBuffer.lhs`.

Reply:
yes.